.travis.yml

```diff
@@ -18,14 +18,9 @@ matrix:
     - php: nightly
       env: DEPS=dev
 
-    - php: 7.0
+    - php: 7.1.3
       env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_PHPUNIT_VERSION=5.7 SYMFONY_DEPRECATIONS_HELPER=23
 
-    - php: 7.1
-      env: LTS_VERSION=2 SYMFONY_DEPRECATIONS_HELPER=23
-    - php: 7.1
-      env: LTS_VERSION=3
```

Comment:
Why did you remove these two? Is there something that blocks support for Symfony 2.8 and 3.4?